runtime/worker.rs

```diff
@@ -214,6 +214,8 @@ pub struct WorkerServiceOptions<
 
   /// V8 code cache for module and script source code.
   pub v8_code_cache: Option<Arc<dyn CodeCache>>,
+
+  pub bundle_provider: Option<Arc<dyn crate::ops::bundle::BundleProvider>>,
```

Comment:
Is it actually optional? What happens if it's None?

Reply:
It falls back to a default implementation that errors out when you call it